Appraisals

```diff
@@ -131,6 +131,7 @@ if ruby_version?('2.1')
     gem 'rake', '< 12.3'
     gem 'rest-client'
     gem 'resque', '< 2.0'
+    gem 'roda', '>= 2.0.0', '< 4.0.0'
```

Comment:
Any particular reason to lock this to `< 4.0.0`.
We don't automatically upgrade gem versions: it has to be manually done. This means someone will supervise any upgrades.
We normally leave gems unlocked, so we can upgrade the easily in the future.

Reply:
I wasn't sure if this instrumentation would work in 4.x so I set a maximum version for now, to avoid a scenario where we say we support 2.0.0+, only for 4.x to have a breaking change.

If this isn't needed at the Appraisal level, I can definitely remove it!

Reply:
I removed the "maximum version" from the latest Appraisal file and ran `bundle exec rake install_appraisal_gemfiles`, but it looks like it changed a lot of files in the gemfiles folder.

Reply:
I just ran `bundle exec rake install_appraisal_gemfiles` today and it didn't change any of the gemfiles so it looks like the gemfiles are already up to date.